.github/workflows/review.yml

```diff
@@ -26,23 +26,23 @@ jobs:
           python-version: ${{ matrix.python }}
 
       - name: Run tests
-        
+
         # Python 3.6 tests have been removed since swagger-spec-validator no longer supports it.
         # A successful workflow run for Python 3.6 is required by the GitHub branch protection rules.
         if: ${{ matrix.python != 3.6 }}
-        
+
         env:
           PYTHON_VERSION: ${{ matrix.python }}
         run: tox -e $(tox -l | grep py${PYTHON_VERSION//.} | paste -sd "," -)
 
       - name: Report coverage
-        if: ${{ matrix.python == 3.9 }}
```

Comment:
Let's bump this up to python3.11 too if we can

Reply:
I believe this is correct as is.  `if: ${{ matrix.python != 3.6 }}` says that we are reporting `coverage` on all versions of Python except the Python 3.6 because `coverage` no longer supports Py36.

Reply:
You're right